File: Baekjoon/2153.cpp
#include <iostream>
using namespace std;

const int MAX = 20 * 52;

int main() {
	bool arr[MAX + 1];
	for (int i = 1; i <= MAX; i++) arr[i] = true;

	for (int i = 2; i * i <= MAX; i++) {
		if (arr[i]) {
			int j = 2;
			while (i * j <= MAX) {
				arr[i * j] = false;
				j++;
			}
		}
	}

	string s;
	int sum = 0;
	cin >> s;
	for (char c : s) {
		if ('a' <= c && c <= 'z') sum += (c - 'a' + 1);
		else sum += (c - 'A' + 27);
	}
	if (arr[sum]) cout << "It is a prime word.\n";
	else cout << "It is not a prime word.\n";
	return 0;
}

File: Baekjoon/2153.py
import math

MAX = 20 * 52

array = [True for _ in range(MAX + 1)]

for i in range(2, int(math.sqrt(MAX)) + 1):
    if array[i]:
        j = 2
        while i * j <= MAX:
            array[i * j] = False
            j += 1
s = input()
sum = 0

for i in range(len(s)):
    if 'a' <= s[i] <= 'z':
        sum += (ord(s[i]) - ord('a')) + 1
    else:
        sum += (ord(s[i]) - ord('A') + 27)
if array[sum]:
    print('It is a prime word.')
else:
    print('It is not a prime word.')
